### PR TITLE
Upload macos packages to s3 during release

### PR DESCRIPTION
**Pull request by dincho**
https://www.pivotaltracker.com/story/show/162785706

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -643,16 +643,7 @@ jobs:
...
-    <<: *infrastructure_config
-    steps:
-      - checkout
-      - attach_workspace:
-          at: *packages_workspace
-      - *setup_secrets
-      - *upload_github_release_asset
-
-  upload_packages_linux:
+  upload_release_packages:
```
Comment (lucafavatella) [inline .circleci/config.yml]: I see step `upload_github_release_asset` has `ASSETS=${PACKAGES_DIR:?}/*` so that is ok.

Comment (lucafavatella) [inline .circleci/config.yml]: How does `upload_release_artifacts` work with one `aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/aeternity-latest-${PKG_SUFFIX:?}.tar.gz`? It feels like it uploads only one package to S3? Is that on purpose?

Comment (dincho) [reply .circleci/config.yml]: Good catch!

Comment (dincho) [reply .circleci/config.yml]: fixed

**Pull request merged**